packages/js/components/src/rating/index.tsx

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
@moon0326 Was this file moved with `git mv`?  It would be good to carry the history of the previous file

Reply:
That's a good point. 

TIL - It looks like Git still preserves the history [here](https://github.com/woocommerce/woocommerce/commits/6567fd11f051e7f7080faaf2fba66effec1e5765/packages/js/components/src/rating/index.tsx). It has `Browse History` link. Is this enough? I can open a new PR otherwise.

Reply:
@adrianduffell Could you please check my last comment? :) I'm wondering if I need to open PRs for other changes if we have to use `git mv`